charts/pulsar/templates/pulsar-manager-deployment.yaml

```diff
@@ -89,6 +89,8 @@ spec:
                 {{- else }}
                 name: "{{ template "pulsar.fullname" . }}-{{ .Values.pulsar_manager.component }}-secret"
                 {{- end }}
+          - name: PULSAR_MANAGER_OPTS
+            value: "$(PULSAR_MANAGER_OPTS) -Dlog4j2.formatMsgNoLookups=true"
```

Comment:
I'm not familiar to this type of syntax, how does `$(PULSAR_MANAGER_OPTS)` behave? can you point to the docs? /cc @michaeljmarshall

Reply:
found the docs, https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/ , it seems.

Reply:
LGTM